Source/items.cpp

```diff
@@ -2253,9 +2257,9 @@ void SetupItem(int i)
 	int it;
 
 	it = ItemCAnimTbl[items[i]._iCurs];
-	items[i]._iAnimData = itemanims[it];
+	items[i]._iAnimData = &*itemanims[it];
 	items[i]._iAnimLen = ItemAnimLs[it];
-	items[i]._iAnimWidth = 96;
+	items[i]._iAnimWidth = ItemAnimWidth;

```

Comment:
Isn't it possible to get this from _iAnimData now?

Reply:
Maybe it is; I've kept this part of the refactoring to the minimum though